src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.windows.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Windows.Storage.Streams;
+using Windows.UI.Input.Inking;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Maui.Platform;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// Drawing view service
+/// </summary>
+public static partial class DrawingViewService
+{
+	/// <summary>
+	/// Get image stream from lines
+	/// </summary>
+	/// <param name="lines">Drawing lines</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="backgroundColor">Image background color</param>
+	/// <returns>Image stream</returns>
+	public static Stream GetImageStream(IList<ILine>? lines,
+		Size imageSize,
+		Color backgroundColor)
+	{
+		if (lines == null)
+		{
+			return Stream.Null;
+		}
+
+		var image = GetImageInternal(lines, backgroundColor);
```

Comment:
and remove the `using` statement at line 35
```suggestion
		using var image = GetImageInternal(lines, backgroundColor);
```